From 6247ca134fb84a609915dfc627c8b3330a681cb5 Mon Sep 17 00:00:00 2001
From: Ray Strode <rstrode@redhat.com>
Date: Fri, 19 Aug 2022 13:55:06 -0400
Subject: local-display-factory: Fix typo in supported session types

There's a typo "tyes" instead of "types" leading to the wrong
supported session types getting set in some cases.

This commit fixes that.

Closes https://gitlab.gnome.org/GNOME/gdm/-/issues/808
---
 daemon/gdm-local-display-factory.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/daemon/gdm-local-display-factory.c b/daemon/gdm-local-display-factory.c
index b3335cd49..8c912cc6d 100644
--- a/daemon/gdm-local-display-factory.c
+++ b/daemon/gdm-local-display-factory.c
@@ -402,7 +402,7 @@ gdm_local_display_factory_create_transient_display (GdmLocalDisplayFactory *fact
                 display = gdm_local_display_new ();
                 g_object_set (G_OBJECT (display),
                               "session-type", session_types[0],
-                              "supported-session-tyes", session_types,
+                              "supported-session-types", session_types,
                               NULL);
                 is_initial = TRUE;
         }
-- 
cgit v1.2.3

